src/components/AddItem.js

```diff
@@ -29,48 +54,82 @@ const AddItem = ({ token, results }) => {
-        id="add-shopping-list-item"
-      />
-      <fieldset>
-        <legend>How soon will you buy this again?</legend>
-        <div>
-          <input
-            onChange={e => setFrequency(parseInt(e.target.value))}
-            type="radio"
+      <Box>
+        <Box>
+          <label htmlFor="add-shopping-list-item">Item Name</label>
+        </Box>
+        <input
+          required
+          value={name}
+          onChange={e => setName(e.target.value)}
+          id="add-shopping-list-item"
+        />
+      </Box>
+      <FormControl component="fieldset">
+        <FormLabel component="legend">
+          <Typography variant="subtitle1">
+            When will you purchase this item?
+          </Typography>
+        </FormLabel>
+        <RadioGroup
+          defaultValue={604800000}
+          aria-label="purchase frequency"
+          name="purchase-frequency"
+          value={frequency}
+          onChange={handleSetFrequency}
+        >
+          <FormControlLabel
+            value={604800000}
+            control={<Radio color="default" />}
+            label="Soon"
             id="soon"
             name="frequency"
-            value={604800000}
-            checked={frequency === 604800000}
+            labelPlacement={'start'}
+            className={classes.formControlLabel}
           />
-          <label htmlFor="soon">Soon</label>
-          <input
-            onChange={e => setFrequency(parseInt(e.target.value))}
-            type="radio"
+          <Typography className={classes.caption} variant="caption">
+            Within 7 days
+          </Typography>
+          <Divider />
+          <FormControlLabel
+            value={1209600000}
+            control={<Radio color="default" />}
+            label="Kind of Soon"
             id="kind-of-soon"
             name="frequency"
-            value={1209600000}
+            labelPlacement="start"
+            className={classes.formControlLabel}
           />
-          <label htmlFor="kind-of-soon">Kind of Soon</label>
-          <input
-            onChange={e => setFrequency(parseInt(e.target.value))}
-            type="radio"
+          <Typography className={classes.caption} variant="caption">
+            More than 7 days, less than 14 days
+          </Typography>
+          <Divider />
+          <FormControlLabel
+            value={2592000000}
+            control={<Radio color="default" />}
+            label="Not Soon"
             id="not-soon"
             name="frequency"
-            value={2592000000}
+            labelPlacement="start"
+            className={classes.formControlLabel}
           />
-          <label htmlFor="not-soon">Not Soon</label>
-        </div>
-      </fieldset>
-      <button type="submit" onClick={() => setName(name.trim())}>
-        Add Item
-      </button>
+          <Typography className={classes.caption} variant="caption">
+            More than 7 days, less than 14 days
```

Comment:
I like how you added the clarifying text!!!  I think for this one you meant more than 14 and less than 30.